upb/bindings/googlepb/bridge.cc

```diff
@@ -94,9 +94,23 @@ const MessageDef* DefBuilder::GetMaybeUnfrozenMessageDef(
     fields.push_back(d->field(i));
   }
 
+  // Oneof fields
+  for (int i = 0, maxi = d->oneof_decl_count(); i < maxi; ++i) {
```

Comment:
Wrap in `#ifdef GOOGLE_PROTOBUF_HAS_ONEOF` so it can compile against protobuf < 2.6.0 (which introduced oneof support).